lib/matplotlib/tests/test_legend.py

```diff
@@ -607,6 +607,21 @@ def test_legend_title_fontprop_fontsize():
     assert leg5.get_title().get_fontsize() == 20
 
 
+@pytest.mark.parametrize('alignment', ('center', 'left', 'right'))
+def test_legend_title_align(alignment):
+    fig, ax = plt.subplots()
+    ax.plot(range(10))
```

Comment:
```suggestion
    ax.plot(range(10), label='test')
```
To make the box non-empty.